tests/foreman/cli/test_host.py

```diff
@@ -3321,3 +3321,23 @@ def test_positive_errata_list_of_sat_server(self):
         hostname = ssh.command('hostname').stdout[0]
         host = Host.info({'name': hostname})
         self.assertIsInstance(Host.errata_list({'host-id': host['id']}), list)
+
+
+class EncDumpTestCase(CLITestCase):
+    """Tests for Dump host's ENC YAML"""
+
+    @tier1
```

Comment:
I am not sure about the tier , as we are not writing tier1 any more, but as this is bug test coverage I have to investigate

Reply:
Please ignore last message this is definitely tier1